src/main/scala/net/ripe/rpki/publicationserver/Boot.scala

```diff
@@ -40,38 +44,50 @@ object Boot extends App {
 
 class PublicationServerApp(conf: AppConfig, logger: Logger) extends RRDPService {
     
-  def run() {
-    XodusDB.init(conf.storePath)
+  implicit val system = ActorSystem.create(Math.abs(new ju.Random().nextLong()).toString())
+  implicit val dispatcher = system.dispatcher
 
-    logger.info("Starting up the publication server ...")
+  var httpBinding: Future[ServerBinding] = _
+  var httpsBinding: Future[ServerBinding] = _
 
-    implicit val system = ActorSystem("0")
-    implicit val materializer = ActorMaterializer()
-    implicit val dispatcher = system.dispatcher
+  def run() {      
+    XodusDB.reset()
+    XodusDB.init(conf.storePath)    
+
+    logger.info("Starting up the publication server ...")    
 
     implicit val timeout = Timeout(5.seconds)
 
     val registry = CollectorRegistry.defaultRegistry
-    val metrics = new Metrics(registry)
+    val metrics = Metrics.get(registry)
     val metricsApi = new MetricsApi(registry)
 
     val stateActor: ActorRef = system.actorOf(StateActor.props(conf, metrics))
 
     val publicationService = new PublicationService(conf, stateActor)
 
-    Http().bindAndHandle(
+    this.httpsBinding = Http().bindAndHandle(
       publicationService.publicationRoutes,
-      interface = "::0",
+      interface = "localhost",
```

Comment:
I remember doing this to pass test on Gitlab. Are we sure we're going to keep it? maybe it's a config parameter?